x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -85,6 +73,29 @@ export function isMappableJob(job: CombinedJob, detectorIndex: number): boolean
   return isMappable;
 }
 
+export function hasValidComposite(buckets: { [key: string]: any }) {
```

Comment:
I think it's important to mention in the comment of this function it checks for `date_histogram` and `terms` only. Maybe worth renaming to reflect the fact it's supported composite for the chart.

Reply:
Updated here [`25af35d` (#96348)](https://github.com/elastic/kibana/pull/96348/commits/25af35d0b966d98edd92a726b6fb74954a3ccc7c)